catalog-rest-service/src/main/resources/json/schema/entity/services/connections/dashboard/tableauConnection.json

```diff
@@ -53,13 +53,14 @@
       "description": "Tableau Environment Name",
       "type": "string"
     },
+    "dbServiceName": {
+      "description": "Database Service Name in order to create a lineage",
```

Comment:
Might feel cleaner to have an EntityReference here to have proper linking 🙏

Reply:
I think sending an EntityReference via JsonConfig might be a little more confusing for users.

Reply:
what dbServiceName allows users to do is, send a serviceName, we create a fqn, using that service name, get the entity's reference and then link it with the dashboard